src/integration-test/integration/ReportsClient.test.ts

```diff
@@ -108,9 +113,8 @@ describe("Reports Client", () => {
   it("Reports - Get all non deleted", async () => {
     const reports = await reportsClient.getReports(accessToken, projectId, undefined, true);
     expect(reports).to.not.be.undefined;
-    for(const report of reports) {
-      expect(["Test1", "Test2", "Test3", "Test"]).to.include(report.displayName);
-    }
+    expect(reports.length).to.be.gt(3);
+    expect(reports.some((x) => x.deleted)).to.be.true;
```

Comment:
So the test is "Get all non deleted" and you expect some reports to be deleted? Makes no sense.

Reply:
renamed test